src/gui/advancedsettings.cpp

```diff
@@ -97,6 +97,7 @@ enum AdvSettingsRows
     OS_CACHE,
     COALESCE_RW,
     SUGGEST_MODE,
+    PIECE_EXTENT_AFFINITY,
```

Comment:
I would prefer move it above `SUGGEST_MODE`.

Reply:
@Chocobo1 just here or everywhere in general, in terms of code organization?

Reply:
I hope it can be applied everywhere (to keep it tidy).

Reply:
@Chocobo1 done